lxd/device/nic_bridged.go

```diff
@@ -382,6 +382,19 @@ func (d *nicBridged) PreStartCheck() error {
 	// Check there isn't another NIC with any of the same addresses specified on the same cluster member.
 	// Can only validate this when the instance is supplied (and not doing profile validation).
 	if d.inst != nil {
+
+		deviceStaticFileName := dnsmasq.StaticAllocationFileName(d.inst.Project(), d.inst.Name(), d.Name())
```

Comment:
Please can you move this after the check below described in the comment above about "Check there isn't another NIC with any of the same addresses specified on the same cluster member." as we shouldn't build the NICs config if it potentially has conflicts with existing config. So move this section to the end of the function please.

Reply:
Done